zipline/protocol.py

```diff
@@ -506,6 +545,26 @@ def has_key(self, name):
         """
         return name in self
 
+    def get_default(self, name):
+        try:
+            sid_data = self[name]
+        except KeyError:
+            sid_data = self[name] = self._siddata_class(name)
+        return sid_data
+
+    def update_sid(self, event):
+        if isinstance(event, WideTradeEvent):
```

Comment:
I might be asking for a premature optimization, but `isinstance` checks have been a bottleneck before.

However, I can see why a separate trade event type is not used, which would be the way to avoid doing this check on every minute.

Thoughts about a boolean in tradesimuaion/algorithm of 'use wide bars?'